Warmup-1/delDel/test/index.test.ts

```diff
@@ -0,0 +1,23 @@
+// Given a string, if the string "del" appears starting at index 1, return a string where that "del" has been deleted. Otherwise, return the string unchanged.
+
+import { assert } from "chai";
+import { delDel } from "../src";
+
+// delDel("adelbc") → "abc"
+// delDel("adelHello") → "aHello"
+// delDel("adedbc") → "adedbc"
+
+describe("delDel", () => {
```

Comment:
Please add a test for when the input is `delabc` and `abcdel`

Reply:
Sure.